.env

```
@@ -57,7 +57,9 @@ SPDK_CENTOS_REPO_VER="9.0-21.el9"
 
 # Ceph Cluster
 CEPH_CLUSTER_VERSION="${CEPH_VERSION}"
-CEPH_VSTART_ARGS="--without-dashboard --memstore"
+# CEPH_CLUSTER_CEPH_REPO_BASEURL= # Uncomment for specifying a custom Ceph YUM repo
```

@baum uncomment and set this to the ceph-ci repo URL:
```suggestion
CEPH_CLUSTER_CEPH_REPO_BASEURL="https://3.chacra.ceph.com/r/ceph/ceph-nvmeof-mon/c1ced49d133c2bcd1fa9311a35688bf641b93c76/centos/9/flavors/default/"
```
You'll need to run `make clean_cache` to prune the build cache.